R/UtilityFunctions.R

```diff
@@ -210,7 +224,7 @@ getFlowsaData <- function(dataname, year) {
 #' @return A dataframe of StateIO data registry on Data Commons.
 getRegistryonDataCommons <- function(data_group = "stateio") {
   registry_ls <- aws.s3::get_bucket(bucket = "dmap-data-commons-ord",
-                                    prefix = data_group)
+                                    prefix = data_group, max = Inf)
```

Comment:
I don't know what the timeline is like for this PR, but if it's going to be more than a few days I might recommend pushing just this line as a fix. stateior is broken for all users right now because the registry is using the default max of 1000; as far as I can tell it is otherwise working.

Reply:
thank you, good catch! If we don't release this soon I will make that fix.